aks/extra-variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "metrics_storage" {
+  description = "Azure Blob Storage configuration values for the storage container where the archived metrics will be stored."
```

Comment:
shouldnt we add `sensitive` here?

Reply:
We will need to better discuss our approach to sensitive secrets, as per our thread in our Slack channel.